roles/openshift_certificate_expiry/test/conftest.py

```diff
@@ -0,0 +1,130 @@
+import pytest
+from OpenSSL import crypto
+
+# TODO: parameterized fixtures for valid/invalid certs
+# TODO: parameterized fixtures for different types of certs
+
+
+@pytest.fixture(scope='session')
+def ca_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp('ca')
```

Comment:
:+1: 

I thought there was a builtin fixture to do just that... but I think I was dreaming, the docs have something like this as an [example only](http://docs.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects).